htdocs/reports/reports_dailypatients.php

```diff
@@ -53,13 +53,31 @@
 	$margin_list[$i] = ($SCREEN_WIDTH * $margin_list[$i] / 100);
 }
 ?>
+<script src="js/jquery.min.js"></script>
+<script src="js/jquery.table2excel.js"></script>
 <script type='text/javascript'>
 function export_as_word(div_id)
 {
 	var content = $('#'+div_id).html();
 	$('#word_data').attr("value", content);
 	$('#word_format_form').submit();
 }
+function export_as_excel(div_id)
+{
+	var content = $('#'+div_id).html();
+
+	const date = new Date();
+	const formattedDate = `${date.getFullYear()}${("0" + (date.getMonth()+1)).slice(-2)}${("0" + (date.getDate())).slice(-2)}${date.getHours()}${date.getMinutes()}`;
+
+	var fileName = `blisreport_${formattedDate}.xls`;
+
+	$("#report_content_table5").table2excel({
+		// exclude: ".excludeThisClass",
+		name: "LangUtil::$generalTerms['PATIENT_DAILY_LOG_SHEET_NAME']",

```

Comment:
I think this is in the Javascript code, so you have to do something like this, right?

```suggestion
		name: "<?php echo(LangUtil::$generalTerms['PATIENT_DAILY_LOG_SHEET_NAME']) ?>",

```